src/index.js

```diff
@@ -3,16 +3,26 @@
 const path = require('path');
 const mapping = require('ember-rfc176-data');
 
-function isBlacklisted(blacklist, importPath, exportName) {
-  if (Array.isArray(blacklist)) {
-    return blacklist.indexOf(importPath) > -1;
+function isDisallowed(disallowlist, importPath, exportName) {
+  if (Array.isArray(disallowlist)) {
+    return disallowlist.indexOf(importPath) > -1;
   } else {
-    let blacklistedExports = blacklist[importPath];
+    let disallowedExports = disallowlist[importPath];
 
-    return blacklistedExports && blacklistedExports.indexOf(exportName) > -1;
+    return disallowedExports && disallowedExports.indexOf(exportName) > -1;
   }
 }
 
+function getDisallowedlist(opts = {}) {
+  if (opts.blacklist) {
+    console.warn(
+      'babel-plugin-ember-modules-api-polyfill: The option "blacklist" is deprecated, use "disallowedlist" instead.'
+    );
+  }
+
+  return opts.disallowlist || opts.blacklist || [];
+}
```

Comment:
Since the only consumer of this is `ember-cli-babel`, and we can release an updated version pretty easily lets just drop support for `blacklist` all together and do a breaking change release.